ParkenDD/Colors.swift

```diff
@@ -90,4 +104,21 @@ extension UIColor {
 		}
 		self.init(red:red, green:green, blue:blue, alpha:alpha)
 	}
+
+    public var rgbaComponents: (red: CGFloat, green: CGFloat, blue: CGFloat, alpha: CGFloat) {
+        var r: CGFloat = 0
+        var g: CGFloat = 0
+        var b: CGFloat = 0
+        var a: CGFloat = 0
+        self.getRed(&r, green: &g, blue: &b, alpha: &a)
+        return (r, g, b, a)
+    }
+    public var hsbaComponents: (hue: CGFloat, saturation: CGFloat, brightness: CGFloat, alpha: CGFloat) {
```

Comment:
If you got this from stack overflow, add a link to the source in the variable's comment description.

Reply:
Added citations and comment descriptions.